lib/error-processor.js

```diff
@@ -65,6 +65,16 @@ class ErrorProcessor {
   }
 }
 
+class ClientTimeoutErrorProcessor extends ErrorProcessor {
+  match(message, err) {
+    return _.includes(message, 'ReadTimeout(');
+  }
+
+  async process(message) {
+    console.log(red(`Fun request api timeout, you can use 'fun config' to set a larger timeout value.`));
```

Comment:
The timeout of API request has been detected，you can increase timeout value by 'fun config'.

Reply:
done